app/scripts/modules/core/src/pipeline/pipeline.states.ts

```diff
@@ -92,5 +93,18 @@ module(PIPELINE_STATES, [APPLICATION_STATE_PROVIDER]).config([
     };
 
     applicationStateProvider.addChildState(pipelines);
+
+    const executionsLookup: INestedState = {
+      name: 'executionLookup',
+      url: '/executions/:executionid',
+      params: {
+        executionid: { dynamic: true },
```

Comment:
why not camel case this?

Reply:
I feel like there was some unspoken rule from the ancient nodejs days where path params were all lowercase?

In other words, idk so let me just camel case it.